packages/analytics-js-plugins/src/utilities/retryQueue/index.ts

```diff
@@ -327,14 +328,12 @@ class Queue extends Emitter {
   }
 
   reclaim(id: string) {
-    const other = new Store(
-      {
-        name: this.name,
-        id,
-        validKeys: QueueStatuses,
-      },
-      getStorageEngine('localStorage'),
-    );
+    const other = this.storeManager.setStore({
+      id,
+      name: this.name,
+      validKeys: QueueStatuses,
+      type: 'localStorage',
```

Comment:
This should also respect the `storageType` input in the constructor.

Reply:
on it

Reply:
seems that this hardcodded value is as expected asreclaim only happens from localstrage